crypto/err/err.c

```diff
@@ -92,17 +93,16 @@ static ERR_STRING_DATA ERR_str_reasons[] = {
     {ERR_R_DSA_LIB, "DSA lib"},
     {ERR_R_X509_LIB, "X509 lib"},
     {ERR_R_ASN1_LIB, "ASN1 lib"},
+    {ERR_R_CRYPTO_LIB, "crypto lib"},
     {ERR_R_EC_LIB, "EC lib"},
     {ERR_R_BIO_LIB, "BIO lib"},
     {ERR_R_PKCS7_LIB, "PKCS7 lib"},
     {ERR_R_X509V3_LIB, "X509V3 lib"},
-    {ERR_R_ENGINE_LIB, "ENGINE lib"},
+    {ERR_R_ENGINE_LIB, "engine lib"},
     {ERR_R_UI_LIB, "UI lib"},
-    {ERR_R_OSSL_STORE_LIB, "STORE lib"},
     {ERR_R_ECDSA_LIB, "ECDSA lib"},
-
-    {ERR_R_NESTED_ASN1_ERROR, "nested asn1 error"},
-    {ERR_R_MISSING_ASN1_EOS, "missing asn1 eos"},
+    {ERR_R_OSSL_STORE_LIB, "store lib"},
+    {ERR_R_OSSL_DECODER_LIB, "decoder lib"},
```

Comment:
?

Reply:
`decoder lib` was missing so far and I moved the two ASN.1 error strings to where they below.
Regarding upper vs. lower case see above.

Reply:
Ah, right, `OSSL ` was missing. Added in both cases.